README.md

```diff
@@ -142,6 +169,81 @@ To change this behavior use `skipOnEmpty(true)`:
 Number::rule()->integer()->max(100)->skipOnEmpty(true)
 ```
 
+#### Nested and related data
+
+```php
+use Yiisoft\Validator\Rule\HasLength;
+use Yiisoft\Validator\Rule\Nested;
+use Yiisoft\Validator\Rule\Number;
+use Yiisoft\Validator\Rule\Required;
+
+$data = ['author' => ['name' => 'Alexey', 'age' => '31']];
+$rule = Nested::rule([
+    'author' => Nested::rule([
+        'name' => [HasLength::rule()->min(3)],
+        'age' => [Number::rule()->min(18)],
+    )];
+]);
+$errors = $rule->validate($data)->getErrors();
+```
+
+A more complex real-life example using arrays.
```

Comment:
```suggestion
A more complex real-life example is a chart that is made of points. This data is represented as an arrays:
```

Reply:
Thanks. Added with small correction and addition. See https://github.com/yiisoft/validator/pull/151#discussion_r793256574. Also fixed a grammar error.

Reply:
:heavy_check_mark: